conf/broker.conf

```diff
@@ -185,6 +185,9 @@ subscriptionRedeliveryTrackerEnabled=true
 # How frequently to proactively check and purge expired subscription
 subscriptionExpiryCheckIntervalInMinutes=5
 
+# Enable Shared subscription (default is enabled)
+subscriptionSharedEnable=true
```

Comment:
Instead of introducing a flag, we should introduce a general setting like `subscriptionTypesEnabled`. If `subscriptionTypesEnabled` is not set, it supports all subscription types. Otherwise, the subscription types that are not on that list will be disabled.